packages/react-instantsearch-hooks-dom/src/ui/CurrentRefinements.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import { cx } from './lib/cx';
+
+type CurrentRefinementCategory = Pick<
+  React.ButtonHTMLAttributes<HTMLButtonElement>,
+  'onClick'
+> & { label: string };
+
+type CurrentRefinement = {
+  label: string;
+  category: CurrentRefinementCategory[];
```

Comment:
_Feel free to disregard this nitpicking comment_: we could pluralize this attribute name as `categories` as it references an array.

Reply:
I used this naming to be fully 1:1 with the matching class name, but we could indeed slightly deviate.

Reply:
Addressed in 87e5c9fc.